### DISCUSSION TITLE
How can the completion activation triggers be configured?

### DISCUSSION THREAD

**Discussion by cergicd** [eclipse/lsp4e#990] [Q&A] [answered]
Hey there,
I have the lsp4e plugin installed and configured a language server with the content type. It works but the completions are triggered with every keystroke... How can this be configured to only activate with e.g. controll+space or on "."? Should that be configurable from the lsp4e plugin or is this more dependent on the language server?

**Comment by rubenporras** [comment, marked as answer]
Hi,

you can take a look at the discussion in https://github.com/eclipse-platform/eclipse.platform.ui/issues/866#issuecomment-1563984962 and more concrete at the PR adding the configuration option: https://github.com/eclipse-platform/eclipse.platform.ui/pull/931
